app/services/nomis_alerts/importer.rb

```diff
@@ -91,7 +101,7 @@ class Importer
       'XB' => :violent,
       'XC' => :escape,
       'XCU' => :escape,
-      'XSDEPORT' => :escape,
+      'XSDEPORT' => :not_to_be_released,
```

Comment:
just flagging this - is it intentional that `XSDEPORT` be remapped from `escape` to `not_to_be_released` ?

Reply:
yes @martyn-w, Jade asked for this chage, it is the spreadsheet
<img width="1130" alt="Screen Shot 2020-07-20 at 14 31 57" src="https://user-images.githubusercontent.com/58971/87943491-0837c800-ca96-11ea-9b92-718ae919febf.png">

Reply:
👍 jolly good